package.json

```diff
@@ -27,14 +27,14 @@
     "css-to-string-loader": "^0.1.3",
     "dateformat": "^3.0.3",
     "file-loader": "^3.0.1",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.1",
     "prettier": "1.18.2",
     "react": "^16.8.6",
```

Comment:
**Nitpick:** Sorry for commenting on this, but `devDependencies` should only include code that is not imported by your code. Just maybe something to think about later.

https://github.com/torchbox/eslint-config-torchbox/blob/b0bce1b2d2552043c1a3b90d2c5d57be5cf0cd5e/config.js#L35-L52

That's usually enforced by TBX eslint configuration.